### PR TITLE
Fix indent in execute_function docstring

### PR DESCRIPTION
**Pull request by johannesgreiner**
### Summary

minor edit: fix indent in execute_function for docu

### Details and comments
docu around `max_credits` was showing up messy before at https://qiskit.org/documentation/apidoc/execute.html (c.f. screenshot)
image (28)

fixed indent

### CODE REVIEW

File: qiskit/execute_function.py [hunk 1]
```diff
@@ -169,9 +169,9 @@ def execute(
             (provided the backend supports it). For OpenPulse jobs, only
             measurement level 2 supports this option. Default: False
 
-        max_credits (int): DEPRECATED This parameter is deprecated as of
-        Qiskit Terra 0.20.0, and will be removed in a future release. This parameter has
-        no effect on modern IBM Quantum systems, and no alternative is necessary.
+        max_credits (int): DEPRECATED This parameter is deprecated as of Qiskit Terra 0.20.0
+            and will be removed in a future release. This parameter has no effect on modern
```
Comment (mtreinish) [inline qiskit/execute_function.py:173]: You can also run `tox -eblack` to run black for code formatting. But I think it's probably just going to remove the trailing whitespace:

```suggestion
            and will be removed in a future release. This parameter has no effect on modern
```

**Review by mtreinish — COMMENTED** [review]
This LGTM, thanks for catching and fixing this. It's failing CI because `black` would reformat the code for you. You can run `tox -eblack` locally to fix the formatting issues. But I left an inline comment because I think this is just a trailing whitespace issue.

**Comment by johannesgreiner** [comment]
thank you Mat @mtreinish . Applied your suggested change, thanks for providing!

**Review by mtreinish — APPROVED** [review]

**Comment by prakharb10** [comment]
I think this too can be fixed:

image

https://github.com/Qiskit/qiskit-terra/blob/186c807877e4af573f1ad8ae6d77bac7a3688e33/qiskit/execute_function.py#L232

```
:math:`m \times n`
```

**Comment by prakharb10** [comment]
> wanna open a new issue/PR since this one is already merged?

Yeah, sounds good.

**Pull request merged**